### PR TITLE
split out journald loading to foreman-proxy-journald package

### PR DESCRIPTION
**Pull request by mmoll**
this should be built into:

* [x] Nightly
* [x] 1.20

### CODE REVIEW

**Comment by mmoll** [comment]
@ekohl next try...

File: debian/bionic/foreman-proxy/foreman-proxy.install [hunk 1]
```diff
@@ -0,0 +1,22 @@
+foreman-proxy                             etc
+bin                                       usr/share/foreman-proxy
+bundler.d/bmc.rb                          usr/share/foreman-proxy/bundler.d
+bundler.d/dhcp_isc.rb                     usr/share/foreman-proxy/bundler.d
+bundler.d/krb5.rb                         usr/share/foreman-proxy/bundler.d
+bundler.d/libvirt.rb                      usr/share/foreman-proxy/bundler.d
+bundler.d/puppetca_token_whitelisting.rb  usr/share/foreman-proxy/bundler.d
+bundler.d/puppet.rb                       usr/share/foreman-proxy/bundler.d
```
Comment (ekohl) [inline debian/bionic/foreman-proxy/foreman-proxy.install]: I wonder if we should drop this. It's only used if Puppet < 3 is used and there are no packages for it on Bionic.

File: debian/stretch/foreman-proxy/foreman-proxy.install [hunk 2]
```diff
@@ -0,0 +1,22 @@
+foreman-proxy                             etc
+bin                                       usr/share/foreman-proxy
+bundler.d/bmc.rb                          usr/share/foreman-proxy/bundler.d
+bundler.d/dhcp_isc.rb                     usr/share/foreman-proxy/bundler.d
+bundler.d/krb5.rb                         usr/share/foreman-proxy/bundler.d
+bundler.d/libvirt.rb                      usr/share/foreman-proxy/bundler.d
+bundler.d/puppetca_token_whitelisting.rb  usr/share/foreman-proxy/bundler.d
+bundler.d/puppet.rb                       usr/share/foreman-proxy/bundler.d
```
Comment (ekohl) [inline debian/stretch/foreman-proxy/foreman-proxy.install]: Same here as on Bionic

**Comment by mmoll** [comment]
> Looks like this already gives the benefit that it no longer ships development and test dependencies.

That was already the case before this change, I removed the respective command from the `rules` file.

File: debian/bionic/foreman-proxy/foreman-proxy.install (hunk 1 above)
Comment (mmoll) [reply debian/bionic/foreman-proxy/foreman-proxy.install]: I guess there might be people out there you built funky special setups to stay on the old version, so I'd rather leave this as is for now.

**Pull request merged**